tests/libraries/math/test_safeMathInt256.py

```diff
@@ -4,6 +4,17 @@
 from ethereum.tools.tester import TransactionFailed
 from pytest import fixture, mark, lazy_fixture, raises
 
+@fixture(scope='session')
+def testerSnapshot(sessionFixture):
+    sessionFixture.uploadAndAddToController('solidity_test_helpers/SafeMathInt256Tester.sol')
```

Comment:
You should be able to just `upload(...)` here, no need to add it to the controller.  Same comment below.